habitat-lab/habitat/tasks/rearrange/sub_tasks/pick_sensors.py

```diff
@@ -182,7 +182,9 @@ def update_metric(self, *args, episode, task, observations, **kwargs):
         self._metric = (
             abs_targ_obj_idx == self._sim.grasp_mgr.snap_idx
             and not self._sim.grasp_mgr.is_violating_hold_constraint()
-            and ee_to_rest_distance < self._config.ee_resting_success_threshold
+            and (
+                ee_to_rest_distance < self._config.ee_resting_success_threshold
+                or self._config.ee_resting_success_threshold == -1.0
```

Comment:
Add comment to this config option in the defaults list that setting it to -1.0 will result in the condition not being checked.